### PR TITLE
Fixed Ireland tax payer number validation regex.

### PR DESCRIPTION
**Pull request by abdallahbeshi**
## Description

Fixed Ireland taxpayer number validation regex.

## Type of change
- [x] Bug fix.
- [ ] Feature.
- [ ] Consolidation.

## Checklist
- [ ] Is breaking change.
- [ ] Documentation updated.
- [ ] Tests included (please specify cases).

### CODE REVIEW

File: src/Core/Mews.Fiscalizations.Core/Model/Countries/Countries.cs [hunk 1]
```diff
@@ -116,7 +116,7 @@ static Countries()
                 Indonesia = NonEuropean("ID"),
                 Iran = NonEuropean("IR"),
                 Iraq = NonEuropean("IQ"),
-                Ireland = European("IE", "^(IE)?[0-9][0-9A-Z\\+\\*][0-9]{5}L$"),
+                Ireland = European("IE", "^(IE)?[0-9A-Z]{8}(?:[0-9A-Z]{1})?$"),
```
Comment (abdallahbeshi) [inline src/Core/Mews.Fiscalizations.Core/Model/Countries/Countries.cs:119]: Source: SII documentation
image

Comment (abdallahbeshi) [reply src/Core/Mews.Fiscalizations.Core/Model/Countries/Countries.cs:119]: Source 2: https://ec.europa.eu/taxation_customs/vies/faqvies.do#item_11
image

**Review by ivanliavonchykmews — DISMISSED** [review]

File: src/Core/Mews.Fiscalizations.Core/Model/Countries/Countries.cs (hunk 1 above)
Comment (KaliCZ) [inline src/Core/Mews.Fiscalizations.Core/Model/Countries/Countries.cs:119]: OK, I accept the change of requirements, but why do we have the non-capturing `?:` in the middle?

Comment (abdallahbeshi) [reply src/Core/Mews.Fiscalizations.Core/Model/Countries/Countries.cs]: Yea i missed that, its redundant.

**Review by KaliCZ — APPROVED** [review]

**Pull request merged**